### PR TITLE
[Aftershock] Mutant npcs to aftershock

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
SUMMARY: Mods "Mutant NPCs to Aftershock"

#### Purpose of change
Aftershock is a world that is teetering between cyberpunk and post-cyberpunk,  Genetic modification, cybernetic modification and uplifted animals are much more common than the base world.  Some mutants will be from blob interactions, some will be from genesplicing, military and civilian experiments, some will be uplifted animals or designed beings that were never human to begin with.  This is a start.  This folder will also eventually have cybernetic NPCs as well.   

#### Describe the solution
Move the Mutant NPCs mod files into Aftershock.  Then make changes as needed.

#### Describe alternatives you've considered
Rewriting all this work.  

#### Testing
Tested in new world creation, mutant NPCs spawn.  Tested in existing world save with Mutant NPCs enabled from prior to obsoletion, no errors and mutant NPCs continue to load.

### CODE REVIEW

File: data/mods/Aftershock/npcs/mutant_npcs/notes.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[
+  {
+
```
Comment (ZhilkinSerg) [inline data/mods/Aftershock/npcs/mutant_npcs/notes.md]:
```suggestion
```

File: data/mods/Aftershock/npcs/mutant_npcs/notes.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    Mutant NPCs in aftershock from obsoleted Mutant NPCs mod
```
Comment (ZhilkinSerg) [inline data/mods/Aftershock/npcs/mutant_npcs/notes.md]:
```suggestion
```

File: data/mods/Aftershock/npcs/mutant_npcs/notes.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[
```
Comment (ZhilkinSerg) [inline data/mods/Aftershock/npcs/mutant_npcs/notes.md]: You probably don't need full `modinfo.json ` from obsolete mod.

**Comment by I-am-Erk** [comment]
One of the previous issues with mutant NPCs was that it invalidates the npc backstory and appearance traits if it popped in. Don't know if that matters or not to you, or even if it's still happening

File: data/mods/Aftershock/npcs/mutant_npcs/notes.md (hunk 3 above)
Comment (Maleclypse) [reply data/mods/Aftershock/npcs/mutant_npcs/notes.md]: So just keep the first sentence or delete the whole file?

**Comment by Maleclypse** [comment]
> One of the previous issues with mutant NPCs was that it invalidates the npc backstory and appearance traits if it popped in. Don't know if that matters or not to you, or even if it's still happening

I'll test for that, and I'm assuming I'd be able to add the `demographics` and `backstory` values back to these NPCs?

File: data/mods/Aftershock/npcs/mutant_npcs/notes.md (hunk 3 above)
Comment (ZhilkinSerg) [reply data/mods/Aftershock/npcs/mutant_npcs/notes.md]: I suggest deleting the whole file.

Comment (Maleclypse) [reply data/mods/Aftershock/npcs/mutant_npcs/notes.md]: Deleted. Thanks.

**Pull request merged**